src/main/java/frameworks_and_drivers/EditMedicineWindow.java

```diff
@@ -75,9 +75,15 @@ public EditMedicineWindow(Scanner scanner, ObservableFrame frame, SelectTimesWin
                 times = selectTimes.getUserInput();
 
                 if (userInput.size() == 5){
-                    userInput.add(timesInput[0].getText());
-                    userInput.add(timesInput[1].getText());
-                    userInput.add(timesInput[2].getText());
+                    if (!checker.isWeekOrDaily(timesInput[0].getText())){
```

Comment:
Since these changes don't work, there is no point in including them. I think making a new commit with the old stuff should get rid of this file.